### PR TITLE
feat: [SRM-14699]: added deeplinks from prometheus metric on service …

### PR DESCRIPTION
**Pull request by karansaraswat19**
…health page

#### Screenshots
Screenshot 2023-05-05 at 5 43 04 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by harshilbits — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/MetricsAndLogs/components/MetricsAnalysisContainer/__tests__/MetricsAnalysisContainer.test.tsx [hunk 1]
```diff
@@ -59,6 +59,22 @@ describe('Unit tests for MetricsAnalysisContainer', () => {
...
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      .mockReturnValue({
+        data: mockedMetricsDataWithDeeplink,
+        loading: false
+      })
+
+    const { container } = render(<WrapperComponent {...props} />)
+
+    // expect(container).toMatchSnapshot()
```
[the remark this answers is not in the document]
Comment (praneshg239) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/MetricsAndLogs/components/MetricsAnalysisContainer/__tests__/MetricsAnalysisContainer.test.tsx:74]: +1

File: src/modules/85-cv/components/TimeseriesRow/TimeseriesRow.tsx [hunk 2]
```diff
@@ -73,9 +75,22 @@ export default function TimeseriesRow({
           <Text color={Color.BLACK} font={FONT_SIZE_SMALL} lineClamp={1}>
             {transactionName}
           </Text>
-          <Text font={FONT_SIZE_SMALL} lineClamp={1}>
-            {metricName}
-          </Text>
+          {metricDeeplinkURL ? (
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/TimeseriesRow/TimeseriesRow.tsx:78]: can be moved to a smaller component

**Review by deepesh-ui — DISMISSED** [review]

**Comment by Nyctophiliac1918** [comment]
retrigger cypress-rest

**Comment by Nyctophiliac1918** [comment]
retrigger cypress-cd

**Pull request merged**